partner_event/hooks.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 David Vidal
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    """Preload proper attendee partner for existing registrations using
+       the same rules the module does"""
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        attendees_emails = env['event.registration'].read_group([
+            ('email', '!=', False),
+            ], ['email'], groupby='email', orderby='email')
```

Comment:
Why ordering by `email`? Is this needed for something?

Reply:
isn't it mandatory to have an `orderby` criteria?

Reply:
No, it's not mandatory.